pl_examples/hydra_examples/pl_template.py

```diff
@@ -20,21 +20,24 @@ def main(cfg: DictConfig):
     # ------------------------
     # 1 INIT LIGHTNING MODEL
     # ------------------------
-    model = LightningTemplateModel(cfg)
+    model = LightningTemplateModel(OmegaConf.masked_copy(cfg, ["data", "model", "scheduler", "opt"]))
```

Comment:
nono..
There shouldn't be anything else in the node that initializes the model.
If you have more there, there is something wrong. you should probably mode all the relevant fields into a sub node.

Reply:
Yeah, so my problem here was I was originally passing in the full config object. Although recent changes to PL, inspect every element of the cfg object and log the value to the logger, hence for MISSING values I am getting errors. So, I made this change to only place  the relevant fields in to the template model. But, just putting it under another group makes more sense.